typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/README.md

```diff
@@ -0,0 +1,24 @@
+## Code Pipeline : Blue/Green deployments to Elastic Beanstalk
```

Comment:
Also please mention in here somewhere that you're using L1 resources.

Reply:
Added comment to README.md